io/disk/scsi_add_remove.py

```diff
@@ -33,44 +37,90 @@ def setUp(self):
         '''
         Function for preliminary set-up to execute the test
         '''
+        self.err_paths = []
+        self.device_list = []
         smm = SoftwareManager()
         if not smm.check_installed("lsscsi") and not smm.install("lsscsi"):
             self.cancel("lsscsi is not installed")
-        self.pci_device = self.params.get("pci_device", default=None)
-        if not self.pci_device:
-            self.cancel("Please provide PCI address for which you \
-                        want to run the test")
+        self.wwids = self.params.get('wwids', default='')
+        self.pci_device = self.params.get("pci_device", default='').split(',')
```

Comment:
The split should happen after checking for presence, similar to self.wwids

Reply:
it works fine even if we have the split at this place. 
So I think it should be fine.

Reply:
@Naresh-ibm 
```
>>> bool(''.split(','))
True
>>> bool('')
False
>>>
```